### PR TITLE
Fix potential NPE from HelixDataAccessor

### PR DESCRIPTION
**Pull request by jackjlli**
When calling `getProperty` method from `HelixDataAccessor`, it could return null.
This PR handles potential NPE for that method in Pinot codebase.

### CODE REVIEW

File: pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java [hunk 1]
```diff
@@ -223,17 +223,19 @@ public void cleanupLLC(final String realtimeTableName) {
...
-    IdealState idealState = HelixHelper.getTableIdealState(_helixManager, realtimeTableName);
     final List<String> segmentsToRemove = new ArrayList<String>();
-    Set<String> allSegments = idealState.getPartitionSet();
-    int removeCount = 0;
-    for (String segmentName : allSegments) {
-      if (SegmentName.isLowLevelConsumerSegmentName(segmentName)) {
-        segmentsToRemove.add(segmentName);
-        removeCount++;
+    IdealState idealState = HelixHelper.getTableIdealState(_helixManager, realtimeTableName);
+    if (idealState != null) {
```
Comment (chenboat) [inline pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java]: NIT for easier reading. 
       if (idealState == null) {
             return; 
        }
        .....

File: pinot-common/src/main/java/org/apache/pinot/common/utils/ServiceStatus.java [hunk 2]
```diff
@@ -385,8 +385,12 @@ public IdealStateAndCurrentStateMatchServiceStatusCallback(HelixManager helixMan
     protected CurrentState getState(String resourceName) {
       PropertyKey.Builder keyBuilder = _helixDataAccessor.keyBuilder();
       LiveInstance liveInstance = _helixDataAccessor.getProperty(keyBuilder.liveInstance(_instanceName));
-      String sessionId = liveInstance.getSessionId();
-      return _helixDataAccessor.getProperty(keyBuilder.currentState(_instanceName, sessionId, resourceName));
+      if (liveInstance == null) {
```
Comment (siddharthteotia) [inline pinot-common/src/main/java/org/apache/pinot/common/utils/ServiceStatus.java]: Would it be appropriate here to raise error (like Preconditions.checkArgument) if it is null?

**Comment by Jackie-Jiang** [comment]
@jackjlli We should drop this pr right?

**Comment by jackjlli** [comment]
Closing this PR since only live instance can be null. Other znode should never be null.
PR to handle NPE from live instance: https://github.com/apache/incubator-pinot/pull/4528

**Pull request closed without merging**